### PR TITLE
Bump the bind version to 9.16.37 to fix CVE-2022-3924, CVE-2022-3094, CVE-2022-3736

### PR DESCRIPTION
**Pull request by BettyRain**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

- Bump the bind version to 9.16.37 to fix CVE-2022-3924, CVE-2022-3094, CVE-2022-3736

###### Change Log  

- - Bump the bind version to 9.16.37

###### Does this affect the toolchain?  

**YES*

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-3924
- https://nvd.nist.gov/vuln/detail/CVE-2022-3094
- https://nvd.nist.gov/vuln/detail/CVE-2022-3736

###### Test Methodology

- Pipeline build id: [418733](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=418733&view=results)

### CODE REVIEW

File: SPECS/bind/bind.spec [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 Summary:        Domain Name System software
 Name:           bind
-Version:        9.16.33
-Release:        2%{?dist}
+Version:        9.16.37
+Release:        3%{?dist}
```
Comment (mfrw) [inline SPECS/bind/bind.spec:13]: This should be `1` if we bump the version :)
```suggestion
Release:        3%{?dist}
```

File: SPECS/bind/bind.spec [hunk 2]
```diff
@@ -615,6 +615,9 @@ fi;
 %{_mandir}/man8/named-nzd2nzf.8*
 
 %changelog
+* Thu Sep 7 2023 Betty Lakes <bettylakes@microsoft.com> - 9.16.37-3
```
Comment (mfrw) [inline SPECS/bind/bind.spec:618]:
```suggestion
* Thu Sep 07 2023 Betty Lakes <bettylakes@microsoft.com> - 9.16.37-1
```

**Review by mfrw — APPROVED** [review]
LGTM :)
A few cosmetic fixes requested though

File: SPECS/bind/bind.spec (hunk 1 above)
Comment (BettyRain) [reply SPECS/bind/bind.spec]: Thank you for catching that!

**Pull request merged**